docs/sources/flow/reference/components/pyroscope.ebpf.md

```diff
@@ -0,0 +1,245 @@
+---
+title: pyroscope.ebpf
+labels:
+  stage: beta
+---
+
+# pyroscope.ebpf
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`pyroscope.ebpf` configures an ebpf profiling job for the current host. The collected performance profiles are forwarded
+to
+the list of receivers passed in
+`forward_to`.
+
+You can specify multiple `pyroscope.ebpf` components by giving them different labels, however it is not recommended.
+
+## Usage
+
+```river
+pyroscope.ebpf "LABEL" {
+  forward_to = RECEIVER_LIST
```

Comment:
good catch.

It `targets` used to be not required at the beginning - may be omitted if used with `targets_only=false && default_target={...}` .
Then we made it required and removed `targets_only` and `default_target` arguments
And now it is required (but only in documentation) and we also returned back `targets_only` and `default_target` arguemnts, but we do not document them and do not commit to support them. This may be useful if someone want's to profile the whole system, not only pods. This is "temporary"  :)  until I implement other service discovery options to profile specific processes/systemd units or something else.

I've added targets to the example as you suggested. Thanks.